miranda/convert/_data_definitions.py

```diff
@@ -258,12 +260,12 @@ def gather_raw_rdrs_by_years(
     path = Path(path)
     year_sets = dict()
     for year in range(1950, datetime.datetime.now().year + 1):
-        files = sorted(list(path.glob(f"*_{year - 1}12*.nc")))
+        files = sorted(list(path.glob(f"{year - 1}12*.nc")))
         if files:
             files = [files[-1]]
-        files.extend(sorted(list(path.glob(f"*_{year}*.nc"))))
+        files.extend(sorted(list(path.glob(f"{year}*.nc"))))
         year_sets[str(year)] = files
-    return {"rdrs-v21": year_sets}
+    return {"rdrs-v2.1": year_sets}
```

Comment:
The reason why I removed the period is that this string is used to create the filename. Would we want to have some logic that cleans that out before writing filenames?

Reply:
If we don't want the period I guess we should remove it everywhere to avoid confusion?  Data is running / writing ok with the "v2.1"  is it going to break something in xscen or miranda elsewhere?

Reply:
Likely. I don't think we use `str().split(".")` anywhere, but there are some `Path().suffix` calls. Need to read up on the default behaviour (whether it takes the last `.` or everything after the first).

Reply:
I think I can live with `v21` it will just be cleaner / safer in my mind